tensorflow_datasets/core/features/__init__.py

```diff
@@ -19,6 +19,7 @@
 
 """
 
+from tensorflow_datasets.core.features.array_2d import Array2D
```

Comment:
We do not want to add to the public API something which is not working. `Array2D` should be declared directly in `hugging_facewrapper`.